test/user.js

```diff
@@ -3,10 +3,18 @@ const chai = require('chai'),
   server = require('./../app'),
   logger = require('../app/logger'),
   User = require('../app/models').Users,
+  factory = require('factory-girl').factory,
   token = require('../app/services/tokenSessions'),
   should = chai.should(),
   expect = require('chai').expect;
 
+factory.define('user', User, {
```

Comment:
Set this in other file, in that file you can define all models for all test files and share the same factory.